core/src/main/java/ru/urururu/sanity/CallsMap.java

```diff
@@ -91,4 +82,9 @@ public void visit(Switch switchElement) {
     public void visit(NoOp noOp) {
 
     }
+
+    @Override
+    public void visit(Return returnStatement) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1186)